src/utils/handleRequestWith.ts

```diff
@@ -57,6 +57,12 @@ export const handleRequestWith = (
 
       // Ignore irrelevant worker message types
       if (type !== 'REQUEST') {
+        if (type === 'KEEPALIVE_RESPONSE' && !options.quiet) {
+          console.info(
+            `[MSW] Received Keep Alive response from the mockServiceWorker`,
```

Comment:
This is going to considerably clutter the developer's console. Maybe let's leave out the logging of internal communications?

Reply:
Sounds good. removed.